meson.build

```diff
@@ -44,13 +45,12 @@ else
 endif
 
 jsoncpp_lib = library(
-  'jsoncpp',
-  [ jsoncpp_headers,
-    'src/lib_json/json_tool.h',
```

Comment:
i am new at meson.
just make sure if this file `json_tool.h` will be removed or a mistake, because there are some methods used in `json_writer.cpp` and `json_reader.cpp`

Reply:
a (compiled) target doesn't need to include `.h` files. `.h` files aren't compiled and hence don't have to be part of the target.